extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -168,6 +168,22 @@ function createCompletionItem(commandLine: string, cursorPosition: number, prefi
 	};
 }
 
+async function isExecutable(filePath: string): Promise<boolean> {
+	if (!osIsWindows()) {
```

Comment:
Inverting the if and giving a short comment about how it's different there, why we don't use PATHEXT, etc. would be helpful for future us